main.js

```diff
@@ -30,5 +30,29 @@ client.on('ready', () => {
         }
     });
 });
+client.on('message', message => { //Ten event wykonuje się, gdy bot wykryje wiadomość
```

Comment:
Tak ogólnie, to przeniósłbym ciało tego event handlera do osobnej nazwanej funkcji, np. `handleEveryoneAndHereCommands` (która mogła by być importowana z osobnego modułu), bo niewykluczone, że dojdzie w tym handlerze więcej kodu, a wtedy - przy obecnej formie - zrobi się bałagan.